service/observe/subscription.go

```diff
@@ -155,6 +187,10 @@ func (s *Subscriber) WatchLoop(watch WatchObjects) {
 
 	defer GetInstance().GetBroker().Unsubscribe(s.TopicName, s)
 
+	// initialise the metrics, as there's no rate change if it just pops up with
+	// the value of 1.
+	s.initMetrics(watch.Job.GetNamespace(), string(watch.JobName))
```

Comment:
This gets executed over and over again but needs to happen only once after the app starts. This better be done within the Subscribers constructor function or a package init function.